cmd/extract.go

```diff
@@ -100,11 +102,18 @@ in preparation to build the final docker image.`,
 			}
 		}
 
+		// Buildah fails if the destination does not exist.
+		Debug.log("Creating extract dir: ", extractDir)
+		err = os.MkdirAll(extractDir, os.ModePerm)
```

Comment:
@gireeshpunathil - this line needs to go in an
```
if (buildah) {
}
```
block, because Docker doesn't work with a directory that already exists. 
Can you please make the change and resubmit? Thanks.

Reply:
@chilanti - done, thanks.